wiki/Manual.md

```diff
@@ -333,14 +333,14 @@ Every exchange has a set of properties and methods, most of which you can overri
 
 Here's an overview of generic exchange properties with values added for example:
 
-```javascript
+```json
 {
-    'id':   'exchange'                   // lowercase string exchange id
-    'name': 'Exchange'                   // human-readable string
+    'id':   'exchange',                  // lowercase string exchange id
```

Comment:
JSON does not support single-quoted strings, but aside from that – does this edit add actual value to the documentation?